### ISSUE TITLE
hover breaks other tools

### ISSUE THREAD

**Issue by mjkvaak** [ioam/holoviews#2901]
Hi, I noticed that after hv and bokeh to the newest available versions the hover tool makes other tools jammed when using chrome so that selecting them from the toolbar works only every now and then. The problem did not exist with the older version I used to have (bokeh '0.12.13', hv from half a year ago). The problem can even be verified in hv's examples at the project's websites, see eg. http://holoviews.org/user_guide/Network_Graphs.html and try clicking on the zoom tools. The same problem did not exist on firefox.

**Comment by philippjfr** [comment]
Can't seem to reproduce this in Chrome at least but we should probably raise this upstream with bokeh if this continues to be an issue.

**Comment by jlstevens** [comment]
Just to say I played around with the tools on the page you linked to and I am also unable to reproduce this issue using Chrome. 

From your description, it sounds like a bokeh issue to me so if you have a reproducible example (e.g maybe it is a problem with a specific bokeh/chrome version or OS?), then that might be the best place to report this problem.

One other thought - you might want to consider posting a GIF here to demonstrate the bad behavior you are experiencing...

**Comment by jbednar** [comment]
I can't reproduce using Mac OS X Chrome Version 67.0.3396.99 (Official Build) (64-bit).  I think it must be a chrome- or OS-version-specific problem.

**Comment by mjkvaak** [comment]
Yeah, sure: I'm running on Win 7, SP1, 64-bit and Chrome Version 67.0.3396.99 (Official Build) (64-bit). Same problem existed also on the newer Chrome Version 68.0.3440.84 (Official Build) (64-bit). The problem might be related to the hardware, since the computers at work, though having plenty of processing power, only have mobo integrated graphics cards.

**Comment by jbednar** [comment]
I just tested on a Windows 10 machine using Chrome 67.0.3396.99, and I wasn't able to reproduce any problem.  This machine was a not very powerful laptop from a few years ago, probably also with an integrated graphics card, so apart from the Windows version it would seem comparable.  I guess I'll close this as not reproducible, but it would be great to have a screencast showing precisely what the problem is in case someone else experiences it.

**Comment by mjkvaak** [comment]
Hi! Sorry to have caused the trouble by reporting a bug that no one else seems to be suffering from - luckily. I attached a short video showing the laggish behavior of the toolbar I am experiencing when hover is enabled, ie I need to click a tool multiple times to make it active. With previous chrome version it was even worse: sometimes the buttons were completely unresponsive. When hover is disabled, the tools work as they should.
![6ttrs-dspp2](https://user-images.githubusercontent.com/37109302/43819128-5ba10534-9aea-11e8-85f9-ad3e0af37a38.gif)

**Comment by jbednar** [comment]
Thanks for the screencast!  Now I finally see the problem!  I was able to reproduce it on Chrome Version 68.0.3440.84 (Official Build) (64-bit) for the plot http://holoviews.org/user_guide/Network_Graphs.html#Supplying-explicit-paths . Specifically, some tools could be selected first try, but I had to click on others three or four times before they would activate.  I didn't notice any that would *never* activate, but some definitely took several clicks.
